domains/blog.edwinsanjosoji.json

```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "edwinsanjo",
+        "email": "edwinsanjosoji@gmail.com"
+    },
+    "records": {
+        "CNAME": "hashnode.network"
+    },
+  "TXT": ["_vercel","vc-domain-verify=blog.edwinsanjosoji.is-a.dev,72f6196c4f1373e4be50"]
+}
```

Comment:
This JSON is not valid. The TXT should be located inside the CNAME when you processed this.